src/test/resources/draft2020-12/issue495.json

```diff
@@ -1,61 +1,105 @@
 [
     {
         "description": "issue495 using ECMA-262",
-        "regex": "ecma-262",
         "schema": {
             "$schema": "https://json-schema.org/draft/2020-12/schema",
-            "pattern": "^[a-z]{1,10}$",
+            "patternProperties": {
+              "^[a-z]{1,10}$": true,
+              "(^1$)": true
+            },
```

Comment:
The `pattern` keyword didn't seem to work, so I switched to `patternProperties`; let me know if something else was intended. Also added an extra test case to show why inspecting the regex will not work in general.